### DISCUSSION TITLE
add Dynamic shop support

### DISCUSSION THREAD

**Discussion by masood151117** [BG-Software-LLC/WildChests#188] [Approved Feature Requests]
please add Dynamicshop support to sell chests and wildtools sellwands . https://www.spigotmc.org/resources/dynamicshop-3-1-16-5-1-19-4.65603/
this plugin is much more better than any other shop plugins

**Comment by Kristupas122** [comment]
Agree, just what i need :)

**Comment by OmerBenGera** [comment]
This plugin is flagged as malicious by virustotal and bitdefender.
I don't recommend using it
